app/src/main/java/de/danoeh/antennapod/activity/gpoddernet/GpodnetAuthenticationActivity.java

```diff
@@ -123,6 +125,11 @@ public void onClick(View v) {
                 final String usernameStr = username.getText().toString();
                 final String passwordStr = password.getText().toString();
 
+                if(usernameHasUnwantedChars(usernameStr)){
```

Comment:
Please add spaces after `if` and before `{`

Reply:
Let me make the change.

Reply:
I have made the necessary changes here.